pallets/crowdloan-reward/Cargo.toml

```diff
@@ -41,9 +42,11 @@ default = ['std']
 runtime-benchmarks = ["frame-benchmarking"]
 std = [
   'codec/std',
+  'sp-std/std',
+  'trait-crowdloan-reward/std',
   'frame-support/std',
   'frame-system/std',
-  'log/std',
+  'log',
```

Comment:
why do we need this crate ?

Reply:
I am not sure. It was there when I took over and I kept it. I believe it is in the substrate template. Never tested if it is there for a reason of some macro using it.